### PR TITLE
Fry something, but a whole bunch of times.

### PR DESCRIPTION
**Pull request by Errorin**
## About The Pull Request
It allows you to fry things twice as a traitor, as it's dastardly to fry something doubly-so.

(This is another goof suggestion)

## Why It's Good For The Game
Sometimes something just needs to be fried more.

## Changelog
:cl:
add: Adds an item to the traitor uplink allowing traitors to fry something after it has been already been fried.
/:cl:

### CODE REVIEW

**Comment by Tharcoonvagh** [comment]
2 tc? You suuree?

**Comment by Errorin** [comment]
> 2 tc? You suuree?

It was going to be one, but now it's double one. I mean if people really want it at 1, then fine.

File: code/game/objects/items/manuals.dm [hunk 1]
```diff
@@ -199,6 +199,15 @@
 				</html>
 			"}
 
+/obj/item/book/manual/chef_recipes/double_fry
```
Comment (Qustinnus) [inline code/game/objects/items/manuals.dm]: This shouldn't be a child of chef_recipes or manuals in general, as these are specifically for just being books with real info. You want to use /obj/item/book/granter

**Comment by Tlaltecuhtli** [comment]
why

**Comment by itseasytosee** [comment]
Its a meme and all but its pretty uncreative considering this already exists.

**Comment by Errorin** [comment]
> Its a meme and all but its pretty uncreative considering this already exists.
Goof conscripted me, I think it's pretty pointless as well, but it seems like he intended to do it at some point.

**Comment by Fikou** [comment]
should cost 20 tc

**Comment by trollbreeder** [comment]
double fried items should deal burn damage to people who eat them

**Comment by Time-Green** [comment]
Or give heart attacks

**Comment by Errorin** [comment]
I have been warned by Floyd from doing too much with this, as upcoming stuff he's doing will break it all anyways.

**Comment by optimumtact** [comment]
this doesn't even make sense as a traitor only ability either

**Comment by Errorin** [comment]
So, I followed what floyd said, made it a granter, etc. I'm not re-upping it because all in in all it's a shite concept overall. This however did come of it.
Absolute state of goofball

**Pull request closed without merging**